code/obj/critter/bee.dm

```diff
@@ -101,7 +101,7 @@
 			if (!istype(M)) return
 			if (prob(20))
 				return CritterAttack(M)
-			if (M.stat || M.getStatusDuration("paralysis"))
+			if (!can_act(M, 0))
```

Comment:
Similarly here (and throughout this file) we have 'is-asleep' checks being changed to 'is-stunned'